examples/wCCD/package_docker.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I would be inclined to change api-helpers to use a version from npm and add
```
        "@types/node": "^18.7.18",
        "fs": "^0.0.1-security",
        "process": "^0.11.10",
        "ts-node": "^10.8.0",
```
to devDependencies in the regular package.json, to not duplicate the files.